pipelines/batch/src/main/java/com/google/fhir/analytics/ParquetMerger.java

```diff
@@ -196,6 +210,50 @@ static List<Pipeline> createMergerPipelines(
 
     Set<String> resourceTypes1 = dwhFiles1.findNonEmptyFhirResourceTypes();
     Set<String> resourceTypes2 = dwhFiles2.findNonEmptyFhirResourceTypes();
+    Set<String> dwhViews1 = new HashSet<>();
```

Comment:
Consider refactoring this method into smaller methods as the size looks a bit big.

Reply:
These changes will be reflected in a separate Incremental Run PR.